iphone/cli/hooks/run.js

```diff
@@ -13,14 +13,24 @@ const appc = require('node-appc'),
 	ioslib = require('ioslib'),
 	i18n = appc.i18n(__dirname),
 	__ = i18n.__,
-	__n = i18n.__n;
+	__n = i18n.__n,
+	exec = require('child_process').exec;
+
 
```

Comment:
<!--
  1 failure:  iphone/cli/hooks/...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/run.js;
  Line: 19;
-->
- :no_entry_sign: iphone/cli/hooks/run.js line 19 – More than 1 blank line not allowed. (no-multiple-empty-lines)